src/app/components/OperatorsList/OperatorsList.tsx

```diff
@@ -104,10 +104,13 @@ const OperatorsList = () => {
             {row.map((cell: any, cellIndex: number) => {
                 return (
                   <Grid key={`cell-key-${cellIndex}`} xs={cellIndex > 0 ? 4 : 12}>
-                    <StyledCell
-                      key={`cell-${cellIndex}`}
-                      >
-                      <Typography className={classes.TableCellLabel}>{OPERATOR_CELL_LABEL_NAME[cellIndex]}</Typography>
+                    <StyledCell key={`cell-${cellIndex}`}>
+                      <Typography className={classes.TableCellLabel}>{OPERATOR_CELL_LABEL_NAME[cellIndex]}{OPERATOR_CELL_LABEL_NAME[cellIndex] === 'Status' && (
```

Comment:
This is not transparent logic, can you make it more clear what is cell label name under index and why you can't use some readable labels here?